src/containers/ListPageLayout/ListPageLayout.js

```diff
@@ -72,6 +88,11 @@ export const ListPageLayout = ({
     setPath(newURL);
   }
 
+  const resourcesForCurrentPage = resources.slice(
+    (page - 1) * pageSize,
+    (page - 1) * pageSize + pageSize
```

Comment:
Could simplify to `page * pageSize` for the second value?

Reply:
absolutely 🥇

Reply:
Done